package.json

```diff
@@ -44,9 +44,11 @@
     "@edx/paragon": "^2.5.6",
     "classnames": "^2.2.5",
     "prop-types": "^15.6.1",
+    "universal-cookie": "^2.1.2"
+  },
+  "peerDependencies": {
     "react": "^16.3.0",
     "react-dom": "^16.3.0",
```

Comment:
Trailing comma for last element in `JSON` object will result in a syntax error, I think.